data_centre/data.py

```diff
@@ -70,10 +70,12 @@ def __init__(self, message_handler):
         if os.path.isfile(self.PATH_TO_DATA_OBJECTS + self.SHADER_BANK_DATA_JSON):
             self.shader_bank_data = self._read_json(self.SHADER_BANK_DATA_JSON)
         self.settings = self._read_json(self.DEFAULT_SETTINGS_JSON)
+        self.default_settings = self.settings.copy()
 
         if os.path.isfile(self.PATH_TO_DATA_OBJECTS + self.SETTINGS_JSON):
             self.settings = self._read_json(self.SETTINGS_JSON)
-            self.settings['user_input']['REMOTE_SERVER']['value'] = 'disabled' # remote server off at boot
```

Comment:
agreed it is better to safely access all these things , but in general all the settings expected in the code should be in the config - whenever i am adding new things and get key errors like this i just delete my settings.json file , then on next boot it will create a new file from the settings_defaults.json with everything that the code expects

also actually just released we (i) shouldnt set settings values directly to the dict like this - it is better to use the data method - https://github.com/langolierz/r_e_c_u_r/blob/dev/data_centre/data.py#L217 - since it will also persist the setting (when someone unplugs for example ...)